library/unwind/src/lib.rs

```diff
@@ -68,5 +91,5 @@ extern "C" {}
 extern "C" {}
 
 #[cfg(all(target_vendor = "fortanix", target_env = "sgx"))]
-#[link(name = "unwind", kind = "static-nobundle")]
+#[link(name = "unwind", kind = "static")]
```

Comment:
The PR description says

>x86_64-fortanix-unknown-sgx don't link libunwind built in build-script into rlib

but you *are* linking (*) the libunwind.a into the rlib here instead.

---
(*) Or rather "archiving"/"bundling".

Reply:
I mean I will fix x86_64-fortanix-unknown-sgx not linking libunwind.a into rlib